.gitignore

```diff
@@ -1,3 +1,4 @@
 *.dol
 *.elf
 build/
+bin/
```

Comment:
i forgot, why the ignore add for bin?

Reply:
Because otherwise my vscode want's to push an empty bin folder, and the bin folder just contains built examples, no use for them being pushed out to the repo.

Reply:
i'd remove that then, as the .vscode needs to be deleted as well. i know vscode will try to push it, but afaik you can tell it to not include it in the commit (or is vscode's git integration that bad?)

Reply:
When you run the Makefile in the root of wii-examples, it builds everything, and then makes a bin folder, and copies every built DOL to there. I'll add .vscode to gitignore just so it doesn't try.